File: src/main/java/com/wafflestudio/draft/security/oauth2/client/exception/OAuthTokenNotValidException.java
package com.wafflestudio.draft.security.oauth2.client.exception;

import org.springframework.security.core.AuthenticationException;

public class OAuthTokenNotValidException extends AuthenticationException {
    public OAuthTokenNotValidException(String msg, Throwable t) {
        super(msg, t);
    }

    public OAuthTokenNotValidException(String msg) {
        super(msg);
    }

}

File: src/main/kotlin/com/wafflestudio/draft/security/oauth2/client/exception/OAuthTokenNotValidException.kt
package com.wafflestudio.draft.security.oauth2.client.exception

import org.springframework.security.core.AuthenticationException

class OAuthTokenNotValidException : AuthenticationException {
    constructor(msg: String?, t: Throwable?) : super(msg, t) {}
    constructor(msg: String?) : super(msg) {}
}